charts/hermes-relayer/templates/deployment.yaml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "hermes-relayer.fullname" . }}
+  namespace: {{ include "hermes-relayer.namespace" . }} 
+  labels:
+    {{- include "hermes-relayer.labels" . | nindent 4}}
+spec:
+  replicas: {{.Values.replicaCount}}
+  selector:
+    matchLabels:
+      {{- include "hermes-relayer.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "hermes-relayer.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.podSecurityContext }}
+      securityContext:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      volumes:
+      - name: chain1mnenomic
+        secret:
+          secretName: {{ .Values.relayer.chain1SecretName }}
+      - name: chain2mnenomic
+        secret:
+          secretName: {{ .Values.relayer.chain2SecretName }}
+      - name: keys
+        emptyDir: {}
+      - name: configfile
+        configMap:
+          name: {{ .Release.Name }}-configmap
+          items:
+            - key: config.toml
+              path: config.toml
+      initContainers:
+        - name: initchain1
+          image: "{{.Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion}}"
+          args: [
+            'keys',
+            'add',
+            '--chain',
+            '{{ .Values.relayer.chain1ID }}',
+            '--mnemonic-file',
+            '/etc/mnemonic/mnemonic'
+          ]
+          volumeMounts:
+            - name: keys
+              mountPath: "/home/hermes/.hermes/keys"
+            - name: configfile
+              mountPath: "/home/hermes/.hermes/"
+            - name: chain1mnenomic
+              readOnly: true
+              mountPath: /etc/mnemonic
+        - name: initchain2
+          image: "{{.Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion}}"
+          args: [
+            'keys',
+            'add',
+            '--chain',
+            '{{ .Values.relayer.chain2ID }}',
+            '--mnemonic-file',
+            '/etc/mnemonic/mnemonic'
+          ]
+          volumeMounts:
+            - name: keys
+              mountPath: "/home/hermes/.hermes/keys"
+            - name: configfile
+              mountPath: "/home/hermes/.hermes/"
+            - name: chain2mnenomic
+              readOnly: true
+              mountPath: /etc/mnemonic
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{.Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion}}"
+          imagePullPolicy: {{.Values.image.pullPolicy}}
+          {{- with .Values.containerSecurityContext }}
```

Comment:
it should be move before https://github.com/archway-network/helm-charts/pull/3/files#diff-eea9971305cd2a5fcd75b1a4355582ae64d8e634615aee0179a0be79a2f150bdR87